app/test/testformeditors.cpp

```diff
@@ -0,0 +1,342 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "testformeditors.h"
+
+#include <QtTest/QtTest>
+#include <memory>
+
+void TestFormEditors::init()
+{
+}
+
+void TestFormEditors::cleanup()
+{
+}
+
+void TestFormEditors::testDoublesValidation()
+{
+//  std::unique_ptr<InputNumberValidator> validator = std::unique_ptr<InputNumberValidator>( new InputNumberValidator() );
```

Comment:
All these tests were here for the validator I created at first. But since we are not using it, I will change these tests for attribute controller.